var/spack/repos/builtin/packages/ascent/package.py

```diff
@@ -246,10 +249,10 @@ def create_host_config(self, spec, prefix):
             cfg.write("# python from spack \n")
             cfg.write(cmake_cache_entry("PYTHON_EXECUTABLE",
                       spec['python'].command.path))
-            # install module to standard style site packages dir
-            # so we can support spack activate
-            cfg.write(cmake_cache_entry("PYTHON_MODULE_INSTALL_PREFIX",
-                                        site_packages_dir))
+            # only set dest python site packages dir if passed
+            if py_site_pkgs_dir is not None:
```

Comment:
Would you be using `py_site_pkgs_dir` if it were `''`? In other words, if `py_site_pkgs_dir` would always be a nonempty string when you intend to use it, you can say `if py_site_pkgs_dir:`.

Reply:
Sure thing, happy to do this. 

Sorry this my general python style is leaking though.